lib/workers/global/config/parse/file.ts

```diff
@@ -3,15 +3,15 @@ import { migrateConfig } from '../../../../config/migration';
 import type { AllConfig } from '../../../../config/types';
 import { logger } from '../../../../logger';
 
-export function getConfig(env: NodeJS.ProcessEnv): AllConfig {
+export async function getConfig(env: NodeJS.ProcessEnv): Promise<AllConfig> {
   let configFile = env.RENOVATE_CONFIG_FILE || 'config';
   if (!upath.isAbsolute(configFile)) {
     configFile = `${process.cwd()}/${configFile}`;
     logger.debug('Checking for config file in ' + configFile);
   }
   let config: AllConfig = {};
   try {
-    config = require(configFile);
+    config = (await import(configFile)).default;
```

Comment:
Are you sure about the default? Needs real repo run to validate.

Reply:
Initially I didn't add it, but the tests and snapshots were failing as it added the default, so I did basically this:
https://stackoverflow.com/questions/41148057/why-is-typescript-adding-default-to-a-globally-defined-import

Then it went back to green :) So I assumed the behavior matched `require()`, especially considering the SO post. WDYT?

Reply:
I need to validate it, we need to make sure to not break anything.

I maybe can do some debugging tomorrow.

Reply:
I did some testing on my own although I'll let you double check. Here are my results:

Run without config.js, created https://github.com/nejch/test-renovate-import/pull/1:
(`LOG_LEVEL=debug yarn start nejch/test-renovate-import`)
<details>
<pre>
yarn run v1.22.15
$ run-s generate:* 
$ node tools/generate-imports.mjs
generating imports
> data/node-js-schedule.json
> node_modules/emojibase-data/en/shortcodes/github.json
$ node -r ts-node/register/transpile-only -- lib/renovate.ts nejch/test-renovate-import
DEBUG: Using RE2 as regex engine
DEBUG: Parsing configs
DEBUG: Checking for config file in /home/nejc/repos/renovate/config
DEBUG: No config file found on disk - skipping
DEBUG: File config
       "config": {}
DEBUG: CLI config
       "config": {"repositories": ["nejch/test-renovate-import"]}
DEBUG: Env config
       "config": {"hostRules": [], "token": "***********"}
DEBUG: Combined config
       "config": {
         "hostRules": [],
         "token": "***********",
         "repositories": ["nejch/test-renovate-import"]
       }
DEBUG: Using default github endpoint: https://api.github.com/
DEBUG: GitHub 404
       "url": "https://api.github.com/user/emails"
DEBUG: Cannot read user/emails endpoint on GitHub to retrieve gitAuthor
DEBUG: Platform config
       "platformConfig": {"hostType": "github", "endpoint": "https://api.github.com/", "isGhe": false},
       "renovateUsername": "nejch"
DEBUG: Adding token authentication for api.github.com to hostRules
DEBUG: Using baseDir: /tmp/renovate
DEBUG: Using cacheDir: /tmp/renovate/cache
DEBUG: Initializing Renovate internal cache into /tmp/renovate/cache/renovate/renovate-cache-v1
DEBUG: Commits limit = null
DEBUG: Clearing hostRules
DEBUG: Adding token authentication for api.github.com to hostRules
 INFO: Repository started (repository=nejch/test-renovate-import)
       "renovateVersion": "0.0.0-semantic-release"
DEBUG: Using localDir: /tmp/renovate/repos/github/nejch/test-renovate-import (repository=nejch/test-renovate-import)
DEBUG: initRepo("nejch/test-renovate-import") (repository=nejch/test-renovate-import)
DEBUG: Overriding default GitHub endpoint (repository=nejch/test-renovate-import)
       "endpoint": "https://api.github.com/"
DEBUG: nejch/test-renovate-import default branch = main (repository=nejch/test-renovate-import)
DEBUG: Using personal access token for git init (repository=nejch/test-renovate-import)
DEBUG: resetMemCache() (repository=nejch/test-renovate-import)
DEBUG: Resetting npmrc (repository=nejch/test-renovate-import)
DEBUG: detectSemanticCommits() (repository=nejch/test-renovate-import)
DEBUG: Initializing git repository into /tmp/renovate/repos/github/nejch/test-renovate-import (repository=nejch/test-renovate-import)
DEBUG: Performing blobless clone (repository=nejch/test-renovate-import)
DEBUG: git clone completed (repository=nejch/test-renovate-import)
       "durationMs": 1132
DEBUG: latest repository commit (repository=nejch/test-renovate-import)
       "latestCommit": {
         "hash": "ec115fabd73f2789261fc70894175faa98f40dfd",
         "date": "2021-11-15T00:02:24+01:00",
         "message": "Create .nvmrc",
         "refs": "HEAD -> main, origin/main, origin/HEAD",
         "body": "",
         "author_name": "Nejc Habjan",
         "author_email": "hab.nejc@gmail.com"
       }
DEBUG: getCommitMessages (repository=nejch/test-renovate-import)
DEBUG: Semantic commits detection: unknown (repository=nejch/test-renovate-import)
DEBUG: No semantic commits detected (repository=nejch/test-renovate-import)
DEBUG: checkOnboarding() (repository=nejch/test-renovate-import)
DEBUG: isOnboarded() (repository=nejch/test-renovate-import)
DEBUG: findFile(renovate.json) (repository=nejch/test-renovate-import)
DEBUG: findFile(renovate.json5) (repository=nejch/test-renovate-import)
DEBUG: findFile(.github/renovate.json) (repository=nejch/test-renovate-import)
DEBUG: findFile(.github/renovate.json5) (repository=nejch/test-renovate-import)
DEBUG: findFile(.gitlab/renovate.json) (repository=nejch/test-renovate-import)
DEBUG: findFile(.gitlab/renovate.json5) (repository=nejch/test-renovate-import)
DEBUG: findFile(.renovaterc) (repository=nejch/test-renovate-import)
DEBUG: findFile(.renovaterc.json) (repository=nejch/test-renovate-import)
DEBUG: config file not found (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, Configure Renovate, !open) (repository=nejch/test-renovate-import)
DEBUG: Retrieving PR list (repository=nejch/test-renovate-import)
DEBUG: Retrieved 0 Pull Requests (repository=nejch/test-renovate-import)
DEBUG: Found no closed onboarding PR (repository=nejch/test-renovate-import)
DEBUG: Repo is not onboarded (repository=nejch/test-renovate-import)
DEBUG: getBranchPr(renovate/configure) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, open) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, closed) (repository=nejch/test-renovate-import)
DEBUG: Onboarding PR does not exist (repository=nejch/test-renovate-import)
DEBUG: Checking if this org/owner has a default Renovate preset which can be used. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
DEBUG: Failed to retrieve default.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
 INFO: Fallback to renovate.json file as a preset is deprecated, please use a default.json file instead. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: Failed to retrieve renovate.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: Failed to retrieve renovate-config.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: No default org/owner preset found, so the default onboarding config will be used instead. Note: do not be concerned with any 404 messages that preceded this. (repository=nejch/test-renovate-import)
DEBUG: onboarding config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: No renovate config file found (repository=nejch/test-renovate-import)
DEBUG: migrateAndValidate() (repository=nejch/test-renovate-import)
DEBUG: No config migration necessary (repository=nejch/test-renovate-import)
DEBUG: massaged config (repository=nejch/test-renovate-import)
       "config": {}
DEBUG: migrated config (repository=nejch/test-renovate-import)
       "config": {}
DEBUG: Found repo ignorePaths (repository=nejch/test-renovate-import)
       "ignorePaths": ["**/node_modules/**", "**/bower_components/**"]
DEBUG: Using file match: (^|/)tasks/[^/]+\.ya?ml$ for manager ansible (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)galaxy\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: azure.*pipelines?.*\.ya?ml$ for manager azure-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect(-bundle)?\.yml$ for manager batect (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect$ for manager batect-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)WORKSPACE(|\.bazel)$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.bzl$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.?bitbucket-pipelines\.ya?ml$ for manager bitbucket-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: buildkite\.ya?ml for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.buildkite/.+\.ya?ml$ for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Gemfile$ for manager bundler (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.cake$ for manager cake (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Cargo.toml$ for manager cargo (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).circleci/config.yml$ for manager circleci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)cloudbuild.ya?ml for manager cloudbuild (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Podfile$ for manager cocoapods (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)composer.json$ for manager composer (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)deps\.edn$ for manager deps-edn (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)docker-compose[^/]*\.ya?ml$ for manager docker-compose (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/|\.)Dockerfile$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Dockerfile\.[^/]*$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).drone.yml$ for manager droneci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).gitmodules$ for manager git-submodules (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^(workflow-templates|\.github\/workflows)\/[^/]+\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)action\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci-include (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)go.mod$ for manager gomod (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gradle(\.kts)?$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\.properties$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\/.+\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.versions\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)gradle/wrapper/gradle-wrapper.properties$ for manager gradle-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.yaml$ for manager helm-requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)values.yaml$ for manager helm-values (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)helmfile.yaml$ for manager helmfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Chart.yaml$ for manager helmv3 (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^Formula/[^/]+[.]rb$ for manager homebrew (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.html?$ for manager html (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)plugins\.(txt|ya?ml)$ for manager jenkins (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)kustomization\.yaml for manager kustomize (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)project\.clj$ for manager leiningen (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.js$ for manager meteor (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)mix\.exs$ for manager mix (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).node-version$ for manager nodenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.json$ for manager npm (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:cs|fs|vb)proj$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:props|targets)$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)dotnet-tools\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)global\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.nvmrc$ for manager nvm (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)requirements\.(txt|pip)$ for manager pip_requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup.py$ for manager pip_setup (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Pipfile$ for manager pipenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pyproject\.toml$ for manager poetry (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.pre-commit-config\.yaml$ for manager pre-commit (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pubspec\.ya?ml$ for manager pub (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).python-version$ for manager pyenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.ruby-version$ for manager ruby-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.sbt$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: project/[^/]*.scala$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup\.cfg$ for manager setup-cfg (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Package\.swift for manager swift (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.tf$ for manager terraform (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terraform-version$ for manager terraform-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)terragrunt\.hcl$ for manager terragrunt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terragrunt-version$ for manager terragrunt-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^.travis.yml$ for manager travis (repository=nejch/test-renovate-import)
DEBUG: Matched 1 file(s) for manager nvm: .nvmrc (repository=nejch/test-renovate-import)
DEBUG: Found nvm package files (repository=nejch/test-renovate-import)
DEBUG: Found 1 package file(s) (repository=nejch/test-renovate-import)
DEBUG: Need to create onboarding PR (repository=nejch/test-renovate-import)
DEBUG: createOnboardingBranch() (repository=nejch/test-renovate-import)
DEBUG: Checking if this org/owner has a default Renovate preset which can be used. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
DEBUG: Failed to retrieve default.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
 INFO: Fallback to renovate.json file as a preset is deprecated, please use a default.json file instead. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: Failed to retrieve renovate.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: Failed to retrieve renovate-config.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: No default org/owner preset found, so the default onboarding config will be used instead. Note: do not be concerned with any 404 messages that preceded this. (repository=nejch/test-renovate-import)
DEBUG: onboarding config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: Creating onboarding branch (repository=nejch/test-renovate-import)
DEBUG: Committing files to branch renovate/configure (repository=nejch/test-renovate-import)
DEBUG: Setting git author name (repository=nejch/test-renovate-import)
       "gitAuthorName": "Renovate Bot"
DEBUG: Setting git author email (repository=nejch/test-renovate-import)
       "gitAuthorEmail": "renovate@whitesourcesoftware.com"
DEBUG: git commit (repository=nejch/test-renovate-import)
       "deletedFiles": [],
       "ignoredFiles": [],
       "result": {
         "author": null,
         "branch": "renovate/configure",
         "commit": "a255362",
         "root": false,
         "summary": {"changes": 1, "insertions": 3, "deletions": 0}
       }
DEBUG: git push (repository=nejch/test-renovate-import)
       "result": {
         "pushed": [
           {
             "deleted": false,
             "tag": false,
             "branch": true,
             "new": true,
             "alreadyUpdated": false,
             "local": "refs/heads/renovate/configure",
             "remote": "refs/heads/renovate/configure"
           }
         ],
         "branch": {
           "local": "renovate/configure",
           "remote": "renovate/configure",
           "remoteName": "origin"
         },
         "ref": {"local": "refs/remotes/origin/renovate/configure"},
         "remoteMessages": {
           "all": [
             "Create a pull request for 'renovate/configure' on GitHub by visiting:",
             "https://github.com/nejch/test-renovate-import/pull/new/renovate/configure"
           ],
           "pullRequestUrl": "https://github.com/nejch/test-renovate-import/pull/new/renovate/configure"
         }
       }
 INFO: Branch created (repository=nejch/test-renovate-import, branch=renovate/configure)
       "commit": "a255362",
       "onboarding": true
DEBUG: Setting current branch to renovate/configure (repository=nejch/test-renovate-import)
DEBUG: latest commit (repository=nejch/test-renovate-import)
       "branchName": "renovate/configure",
       "latestCommitDate": "2021-11-15T00:04:50+01:00"
DEBUG: Found renovate.json config file (repository=nejch/test-renovate-import)
DEBUG: Repository config (repository=nejch/test-renovate-import)
       "fileName": "renovate.json",
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: migrateAndValidate() (repository=nejch/test-renovate-import)
DEBUG: No config migration necessary (repository=nejch/test-renovate-import)
DEBUG: massaged config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: migrated config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: Found repo ignorePaths (repository=nejch/test-renovate-import)
       "ignorePaths": ["**/node_modules/**", "**/bower_components/**"]
DEBUG: No vulnerability alerts found (repository=nejch/test-renovate-import)
DEBUG: No vulnerability alerts found (repository=nejch/test-renovate-import)
DEBUG: No baseBranches (repository=nejch/test-renovate-import)
DEBUG: extract() (repository=nejch/test-renovate-import)
DEBUG: Setting current branch to main (repository=nejch/test-renovate-import)
DEBUG: latest commit (repository=nejch/test-renovate-import)
       "branchName": "main",
       "latestCommitDate": "2021-11-15T00:02:24+01:00"
DEBUG: Using file match: (^|/)tasks/[^/]+\.ya?ml$ for manager ansible (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)galaxy\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: azure.*pipelines?.*\.ya?ml$ for manager azure-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect(-bundle)?\.yml$ for manager batect (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect$ for manager batect-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)WORKSPACE(|\.bazel)$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.bzl$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.?bitbucket-pipelines\.ya?ml$ for manager bitbucket-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: buildkite\.ya?ml for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.buildkite/.+\.ya?ml$ for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Gemfile$ for manager bundler (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.cake$ for manager cake (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Cargo.toml$ for manager cargo (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).circleci/config.yml$ for manager circleci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)cloudbuild.ya?ml for manager cloudbuild (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Podfile$ for manager cocoapods (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)composer.json$ for manager composer (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)deps\.edn$ for manager deps-edn (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)docker-compose[^/]*\.ya?ml$ for manager docker-compose (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/|\.)Dockerfile$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Dockerfile\.[^/]*$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).drone.yml$ for manager droneci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).gitmodules$ for manager git-submodules (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^(workflow-templates|\.github\/workflows)\/[^/]+\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)action\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci-include (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)go.mod$ for manager gomod (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gradle(\.kts)?$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\.properties$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\/.+\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.versions\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)gradle/wrapper/gradle-wrapper.properties$ for manager gradle-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.yaml$ for manager helm-requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)values.yaml$ for manager helm-values (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)helmfile.yaml$ for manager helmfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Chart.yaml$ for manager helmv3 (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^Formula/[^/]+[.]rb$ for manager homebrew (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.html?$ for manager html (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)plugins\.(txt|ya?ml)$ for manager jenkins (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)kustomization\.yaml for manager kustomize (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)project\.clj$ for manager leiningen (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.js$ for manager meteor (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)mix\.exs$ for manager mix (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).node-version$ for manager nodenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.json$ for manager npm (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:cs|fs|vb)proj$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:props|targets)$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)dotnet-tools\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)global\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.nvmrc$ for manager nvm (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)requirements\.(txt|pip)$ for manager pip_requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup.py$ for manager pip_setup (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Pipfile$ for manager pipenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pyproject\.toml$ for manager poetry (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.pre-commit-config\.yaml$ for manager pre-commit (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pubspec\.ya?ml$ for manager pub (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).python-version$ for manager pyenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.ruby-version$ for manager ruby-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.sbt$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: project/[^/]*.scala$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup\.cfg$ for manager setup-cfg (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Package\.swift for manager swift (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.tf$ for manager terraform (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terraform-version$ for manager terraform-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)terragrunt\.hcl$ for manager terragrunt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terragrunt-version$ for manager terragrunt-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^.travis.yml$ for manager travis (repository=nejch/test-renovate-import)
DEBUG: Matched 1 file(s) for manager nvm: .nvmrc (repository=nejch/test-renovate-import)
DEBUG: Found nvm package files (repository=nejch/test-renovate-import)
DEBUG: Found 1 package file(s) (repository=nejch/test-renovate-import)
 INFO: Dependency extraction complete (repository=nejch/test-renovate-import)
       "baseBranch": "main",
       "stats": {
         "managers": {"nvm": {"fileCount": 1, "depCount": 1}},
         "total": {"fileCount": 1, "depCount": 1}
       }
DEBUG: Package releases lookups complete (repository=nejch/test-renovate-import)
       "baseBranch": "main"
DEBUG: branchifyUpgrades (repository=nejch/test-renovate-import)
DEBUG: 2 flattened updates found: node, node (repository=nejch/test-renovate-import)
DEBUG: Returning 2 branch(es) (repository=nejch/test-renovate-import)
DEBUG: Fetching changelog: https://github.com/nodejs/node (8.13.0 -> v8.17.0) (repository=nejch/test-renovate-import)
DEBUG: Fetching changelog: https://github.com/nodejs/node (8.13.0 -> v16.13.0) (repository=nejch/test-renovate-import)
DEBUG: config.repoIsOnboarded=false (repository=nejch/test-renovate-import)
DEBUG: packageFiles with updates (repository=nejch/test-renovate-import)
       "config": {
         "nvm": [
           {
             "packageFile": ".nvmrc",
             "deps": [
               {
                 "depName": "node",
                 "currentValue": "8.13.0",
                 "datasource": "github-tags",
                 "lookupName": "nodejs/node",
                 "depIndex": 0,
                 "updates": [
                   {
                     "bucket": "non-major",
                     "newVersion": "v8.17.0",
                     "newValue": "8.17.0",
                     "releaseTimestamp": "2019-12-18T18:55:49.000Z",
                     "newMajor": 8,
                     "newMinor": 17,
                     "updateType": "minor",
                     "branchName": "renovate/node-8.x"
                   },
                   {
                     "bucket": "major",
                     "newVersion": "v16.13.0",
                     "newValue": "16.13.0",
                     "releaseTimestamp": "2021-10-26T16:22:38.000Z",
                     "newMajor": 16,
                     "newMinor": 13,
                     "updateType": "major",
                     "branchName": "renovate/node-16.x"
                   }
                 ],
                 "warnings": [],
                 "versioning": "node",
                 "sourceUrl": "https://github.com/nodejs/node",
                 "currentVersion": "8.13.0",
                 "isSingleVersion": true,
                 "fixedVersion": "8.13.0"
               }
             ]
           }
         ]
       }
DEBUG: ensureOnboardingPr() (repository=nejch/test-renovate-import)
DEBUG: getBranchPr(renovate/configure) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, open) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, closed) (repository=nejch/test-renovate-import)
DEBUG: Filling in onboarding PR template (repository=nejch/test-renovate-import)
DEBUG: getConfigDesc() (repository=nejch/test-renovate-import)
DEBUG: getDescriptionArray() (repository=nejch/test-renovate-import)
DEBUG: getScheduleDesc() (repository=nejch/test-renovate-import)
DEBUG: No schedule (repository=nejch/test-renovate-import)
DEBUG: No config description found (repository=nejch/test-renovate-import)
DEBUG: getPrList() (repository=nejch/test-renovate-import)
DEBUG: Creating onboarding PR (repository=nejch/test-renovate-import)
DEBUG: Creating PR (repository=nejch/test-renovate-import)
       "title": "Configure Renovate",
       "head": "nejch:renovate/configure",
       "base": "main",
       "draft": false
DEBUG: PR created (repository=nejch/test-renovate-import, branch=renovate/configure)
       "pr": 1,
       "draft": false
DEBUG: Adding labels '' to #1 (repository=nejch/test-renovate-import)
 INFO: Onboarding PR created (repository=nejch/test-renovate-import)
       "pr": "Pull Request #1"
DEBUG: addAssigneesReviewers(pr=1) (repository=nejch/test-renovate-import)
DEBUG: processRepo() (repository=nejch/test-renovate-import)
DEBUG: Closing Dependency Dashboard (repository=nejch/test-renovate-import)
DEBUG: Retrieving issueList (repository=nejch/test-renovate-import)
DEBUG: Retrieved 0 issues (repository=nejch/test-renovate-import)
DEBUG: Removing any stale branches (repository=nejch/test-renovate-import)
DEBUG: config.repoIsOnboarded=false (repository=nejch/test-renovate-import)
DEBUG: Branch lists (repository=nejch/test-renovate-import)
       "branchList": ["renovate/configure"],
       "renovateBranches": ["renovate/configure"]
DEBUG: remainingBranches= (repository=nejch/test-renovate-import)
DEBUG: No branches to clean up (repository=nejch/test-renovate-import)
DEBUG: Repository timing splits (milliseconds) (repository=nejch/test-renovate-import)
       "splits": {"init": 6380, "extract": 282, "lookup": 16566, "update": 1000},
       "total": 24509
DEBUG: http statistics (repository=nejch/test-renovate-import)
       "urls": {
         "https://api.github.com/graphql (POST)": 3,
         "https://api.github.com/repos/nejch/test-renovate-import/pulls (GET)": 1,
         "https://api.github.com/repos/nejch/test-renovate-import/pulls (POST)": 1,
         "https://api.github.com/repos/nodejs/node (GET)": 1,
         "https://api.github.com/repos/nodejs/node/git/blobs/b71b446039867d0cffee69abe1b27f0b767b9113 (GET)": 1,
         "https://api.github.com/repos/nodejs/node/git/trees/master (GET)": 1,
         "https://api.github.com/repos/nodejs/node/releases (GET)": 1,
         "https://api.github.com/repos/nodejs/node/tags (GET)": 1,
         "https://api.github.com/repositories/27193779/releases (GET)": 2,
         "https://api.github.com/repositories/27193779/tags (GET)": 7
       },
       "hostStats": {"api.github.com": {"requestCount": 19, "requestAvgMs": 427, "queueAvgMs": 0}},
       "totalRequests": 19
 INFO: Repository finished (repository=nejch/test-renovate-import)
       "durationMs": 24509
DEBUG: Renovate exiting
Done in 32.79s.
</pre>
</details>

Run with config.js, created https://github.com/nejch/test-renovate-import/pull/2:
```js
// $ cat custom.js
module.exports = {
    onboardingPrTitle: "Configure (values from config.js)",
    repositories: ['nejch/test-renovate-import'],
  };
```
<details>
<pre>
yarn run v1.22.15
$ run-s generate:* 
$ node tools/generate-imports.mjs
generating imports
> data/node-js-schedule.json
> node_modules/emojibase-data/en/shortcodes/github.json
$ node -r ts-node/register/transpile-only -- lib/renovate.ts nejch/test-renovate-import
DEBUG: Using RE2 as regex engine
DEBUG: Parsing configs
DEBUG: Checking for config file in /home/nejc/repos/renovate/config
DEBUG: File config
       "config": {
         "onboardingPrTitle": "Configure (values from config.js)",
         "repositories": ["nejch/test-renovate-import"]
       }
DEBUG: CLI config
       "config": {"repositories": ["nejch/test-renovate-import"]}
DEBUG: Env config
       "config": {"hostRules": [], "token": "***********"}
DEBUG: Combined config
       "config": {
         "onboardingPrTitle": "Configure (values from config.js)",
         "repositories": ["nejch/test-renovate-import"],
         "hostRules": [],
         "token": "***********"
       }
DEBUG: Using default github endpoint: https://api.github.com/
DEBUG: GitHub 404
       "url": "https://api.github.com/user/emails"
DEBUG: Cannot read user/emails endpoint on GitHub to retrieve gitAuthor
DEBUG: Platform config
       "platformConfig": {"hostType": "github", "endpoint": "https://api.github.com/", "isGhe": false},
       "renovateUsername": "nejch"
DEBUG: Adding token authentication for api.github.com to hostRules
DEBUG: Using baseDir: /tmp/renovate
DEBUG: Using cacheDir: /tmp/renovate/cache
DEBUG: Initializing Renovate internal cache into /tmp/renovate/cache/renovate/renovate-cache-v1
DEBUG: Commits limit = null
DEBUG: Clearing hostRules
DEBUG: Adding token authentication for api.github.com to hostRules
 INFO: Repository started (repository=nejch/test-renovate-import)
       "renovateVersion": "0.0.0-semantic-release"
DEBUG: Using localDir: /tmp/renovate/repos/github/nejch/test-renovate-import (repository=nejch/test-renovate-import)
DEBUG: initRepo("nejch/test-renovate-import") (repository=nejch/test-renovate-import)
DEBUG: Overriding default GitHub endpoint (repository=nejch/test-renovate-import)
       "endpoint": "https://api.github.com/"
DEBUG: nejch/test-renovate-import default branch = main (repository=nejch/test-renovate-import)
DEBUG: Using personal access token for git init (repository=nejch/test-renovate-import)
DEBUG: resetMemCache() (repository=nejch/test-renovate-import)
DEBUG: Resetting npmrc (repository=nejch/test-renovate-import)
DEBUG: detectSemanticCommits() (repository=nejch/test-renovate-import)
DEBUG: Initializing git repository into /tmp/renovate/repos/github/nejch/test-renovate-import (repository=nejch/test-renovate-import)
DEBUG: Performing blobless clone (repository=nejch/test-renovate-import)
DEBUG: git clone completed (repository=nejch/test-renovate-import)
       "durationMs": 1011
DEBUG: latest repository commit (repository=nejch/test-renovate-import)
       "latestCommit": {
         "hash": "ec115fabd73f2789261fc70894175faa98f40dfd",
         "date": "2021-11-15T00:02:24+01:00",
         "message": "Create .nvmrc",
         "refs": "HEAD -> main, origin/main, origin/HEAD",
         "body": "",
         "author_name": "Nejc Habjan",
         "author_email": "hab.nejc@gmail.com"
       }
DEBUG: getCommitMessages (repository=nejch/test-renovate-import)
DEBUG: Semantic commits detection: unknown (repository=nejch/test-renovate-import)
DEBUG: No semantic commits detected (repository=nejch/test-renovate-import)
DEBUG: checkOnboarding() (repository=nejch/test-renovate-import)
DEBUG: isOnboarded() (repository=nejch/test-renovate-import)
DEBUG: findFile(renovate.json) (repository=nejch/test-renovate-import)
DEBUG: findFile(renovate.json5) (repository=nejch/test-renovate-import)
DEBUG: findFile(.github/renovate.json) (repository=nejch/test-renovate-import)
DEBUG: findFile(.github/renovate.json5) (repository=nejch/test-renovate-import)
DEBUG: findFile(.gitlab/renovate.json) (repository=nejch/test-renovate-import)
DEBUG: findFile(.gitlab/renovate.json5) (repository=nejch/test-renovate-import)
DEBUG: findFile(.renovaterc) (repository=nejch/test-renovate-import)
DEBUG: findFile(.renovaterc.json) (repository=nejch/test-renovate-import)
DEBUG: config file not found (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, Configure (values from config.js), !open) (repository=nejch/test-renovate-import)
DEBUG: Retrieving PR list (repository=nejch/test-renovate-import)
DEBUG: Retrieved 1 Pull Requests (repository=nejch/test-renovate-import)
DEBUG: Found no closed onboarding PR (repository=nejch/test-renovate-import)
DEBUG: Repo is not onboarded (repository=nejch/test-renovate-import)
DEBUG: getBranchPr(renovate/configure) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, open) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, closed) (repository=nejch/test-renovate-import)
DEBUG: Found PR #1 (repository=nejch/test-renovate-import)
DEBUG: Onboarding PR does not exist (repository=nejch/test-renovate-import)
DEBUG: Checking if this org/owner has a default Renovate preset which can be used. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
DEBUG: Failed to retrieve default.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
 INFO: Fallback to renovate.json file as a preset is deprecated, please use a default.json file instead. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: Failed to retrieve renovate.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: Failed to retrieve renovate-config.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: No default org/owner preset found, so the default onboarding config will be used instead. Note: do not be concerned with any 404 messages that preceded this. (repository=nejch/test-renovate-import)
DEBUG: onboarding config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: No renovate config file found (repository=nejch/test-renovate-import)
DEBUG: migrateAndValidate() (repository=nejch/test-renovate-import)
DEBUG: No config migration necessary (repository=nejch/test-renovate-import)
DEBUG: massaged config (repository=nejch/test-renovate-import)
       "config": {}
DEBUG: migrated config (repository=nejch/test-renovate-import)
       "config": {}
DEBUG: Found repo ignorePaths (repository=nejch/test-renovate-import)
       "ignorePaths": ["**/node_modules/**", "**/bower_components/**"]
DEBUG: Using file match: (^|/)tasks/[^/]+\.ya?ml$ for manager ansible (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)galaxy\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: azure.*pipelines?.*\.ya?ml$ for manager azure-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect(-bundle)?\.yml$ for manager batect (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect$ for manager batect-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)WORKSPACE(|\.bazel)$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.bzl$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.?bitbucket-pipelines\.ya?ml$ for manager bitbucket-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: buildkite\.ya?ml for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.buildkite/.+\.ya?ml$ for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Gemfile$ for manager bundler (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.cake$ for manager cake (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Cargo.toml$ for manager cargo (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).circleci/config.yml$ for manager circleci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)cloudbuild.ya?ml for manager cloudbuild (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Podfile$ for manager cocoapods (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)composer.json$ for manager composer (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)deps\.edn$ for manager deps-edn (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)docker-compose[^/]*\.ya?ml$ for manager docker-compose (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/|\.)Dockerfile$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Dockerfile\.[^/]*$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).drone.yml$ for manager droneci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).gitmodules$ for manager git-submodules (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^(workflow-templates|\.github\/workflows)\/[^/]+\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)action\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci-include (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)go.mod$ for manager gomod (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gradle(\.kts)?$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\.properties$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\/.+\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.versions\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)gradle/wrapper/gradle-wrapper.properties$ for manager gradle-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.yaml$ for manager helm-requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)values.yaml$ for manager helm-values (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)helmfile.yaml$ for manager helmfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Chart.yaml$ for manager helmv3 (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^Formula/[^/]+[.]rb$ for manager homebrew (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.html?$ for manager html (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)plugins\.(txt|ya?ml)$ for manager jenkins (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)kustomization\.yaml for manager kustomize (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)project\.clj$ for manager leiningen (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.js$ for manager meteor (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)mix\.exs$ for manager mix (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).node-version$ for manager nodenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.json$ for manager npm (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:cs|fs|vb)proj$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:props|targets)$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)dotnet-tools\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)global\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.nvmrc$ for manager nvm (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)requirements\.(txt|pip)$ for manager pip_requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup.py$ for manager pip_setup (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Pipfile$ for manager pipenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pyproject\.toml$ for manager poetry (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.pre-commit-config\.yaml$ for manager pre-commit (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pubspec\.ya?ml$ for manager pub (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).python-version$ for manager pyenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.ruby-version$ for manager ruby-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.sbt$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: project/[^/]*.scala$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup\.cfg$ for manager setup-cfg (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Package\.swift for manager swift (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.tf$ for manager terraform (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terraform-version$ for manager terraform-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)terragrunt\.hcl$ for manager terragrunt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terragrunt-version$ for manager terragrunt-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^.travis.yml$ for manager travis (repository=nejch/test-renovate-import)
DEBUG: Matched 1 file(s) for manager nvm: .nvmrc (repository=nejch/test-renovate-import)
DEBUG: Found nvm package files (repository=nejch/test-renovate-import)
DEBUG: Found 1 package file(s) (repository=nejch/test-renovate-import)
DEBUG: Need to create onboarding PR (repository=nejch/test-renovate-import)
DEBUG: createOnboardingBranch() (repository=nejch/test-renovate-import)
DEBUG: Checking if this org/owner has a default Renovate preset which can be used. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
DEBUG: Failed to retrieve default.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
 INFO: Fallback to renovate.json file as a preset is deprecated, please use a default.json file instead. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: Failed to retrieve renovate.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: Failed to retrieve renovate-config.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: No default org/owner preset found, so the default onboarding config will be used instead. Note: do not be concerned with any 404 messages that preceded this. (repository=nejch/test-renovate-import)
DEBUG: onboarding config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: Creating onboarding branch (repository=nejch/test-renovate-import)
DEBUG: Committing files to branch renovate/configure (repository=nejch/test-renovate-import)
DEBUG: Setting git author name (repository=nejch/test-renovate-import)
       "gitAuthorName": "Renovate Bot"
DEBUG: Setting git author email (repository=nejch/test-renovate-import)
       "gitAuthorEmail": "renovate@whitesourcesoftware.com"
DEBUG: git commit (repository=nejch/test-renovate-import)
       "deletedFiles": [],
       "ignoredFiles": [],
       "result": {
         "author": null,
         "branch": "renovate/configure",
         "commit": "efb5402",
         "root": false,
         "summary": {"changes": 1, "insertions": 3, "deletions": 0}
       }
DEBUG: No file changes detected. Skipping commit (repository=nejch/test-renovate-import)
       "branchName": "renovate/configure",
       "deletedFiles": [],
       "addedModifiedFiles": ["renovate.json"],
       "ignoredFiles": []
DEBUG: Setting current branch to renovate/configure (repository=nejch/test-renovate-import)
DEBUG: latest commit (repository=nejch/test-renovate-import)
       "branchName": "renovate/configure",
       "latestCommitDate": "2021-11-15T00:11:17+01:00"
DEBUG: Found renovate.json config file (repository=nejch/test-renovate-import)
DEBUG: Repository config (repository=nejch/test-renovate-import)
       "fileName": "renovate.json",
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: migrateAndValidate() (repository=nejch/test-renovate-import)
DEBUG: No config migration necessary (repository=nejch/test-renovate-import)
DEBUG: massaged config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: migrated config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: Found repo ignorePaths (repository=nejch/test-renovate-import)
       "ignorePaths": ["**/node_modules/**", "**/bower_components/**"]
DEBUG: No vulnerability alerts found (repository=nejch/test-renovate-import)
DEBUG: No vulnerability alerts found (repository=nejch/test-renovate-import)
DEBUG: No baseBranches (repository=nejch/test-renovate-import)
DEBUG: extract() (repository=nejch/test-renovate-import)
DEBUG: Setting current branch to main (repository=nejch/test-renovate-import)
DEBUG: latest commit (repository=nejch/test-renovate-import)
       "branchName": "main",
       "latestCommitDate": "2021-11-15T00:02:24+01:00"
DEBUG: Using file match: (^|/)tasks/[^/]+\.ya?ml$ for manager ansible (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)galaxy\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: azure.*pipelines?.*\.ya?ml$ for manager azure-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect(-bundle)?\.yml$ for manager batect (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect$ for manager batect-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)WORKSPACE(|\.bazel)$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.bzl$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.?bitbucket-pipelines\.ya?ml$ for manager bitbucket-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: buildkite\.ya?ml for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.buildkite/.+\.ya?ml$ for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Gemfile$ for manager bundler (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.cake$ for manager cake (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Cargo.toml$ for manager cargo (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).circleci/config.yml$ for manager circleci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)cloudbuild.ya?ml for manager cloudbuild (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Podfile$ for manager cocoapods (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)composer.json$ for manager composer (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)deps\.edn$ for manager deps-edn (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)docker-compose[^/]*\.ya?ml$ for manager docker-compose (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/|\.)Dockerfile$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Dockerfile\.[^/]*$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).drone.yml$ for manager droneci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).gitmodules$ for manager git-submodules (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^(workflow-templates|\.github\/workflows)\/[^/]+\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)action\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci-include (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)go.mod$ for manager gomod (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gradle(\.kts)?$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\.properties$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\/.+\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.versions\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)gradle/wrapper/gradle-wrapper.properties$ for manager gradle-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.yaml$ for manager helm-requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)values.yaml$ for manager helm-values (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)helmfile.yaml$ for manager helmfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Chart.yaml$ for manager helmv3 (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^Formula/[^/]+[.]rb$ for manager homebrew (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.html?$ for manager html (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)plugins\.(txt|ya?ml)$ for manager jenkins (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)kustomization\.yaml for manager kustomize (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)project\.clj$ for manager leiningen (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.js$ for manager meteor (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)mix\.exs$ for manager mix (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).node-version$ for manager nodenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.json$ for manager npm (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:cs|fs|vb)proj$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:props|targets)$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)dotnet-tools\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)global\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.nvmrc$ for manager nvm (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)requirements\.(txt|pip)$ for manager pip_requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup.py$ for manager pip_setup (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Pipfile$ for manager pipenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pyproject\.toml$ for manager poetry (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.pre-commit-config\.yaml$ for manager pre-commit (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pubspec\.ya?ml$ for manager pub (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).python-version$ for manager pyenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.ruby-version$ for manager ruby-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.sbt$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: project/[^/]*.scala$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup\.cfg$ for manager setup-cfg (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Package\.swift for manager swift (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.tf$ for manager terraform (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terraform-version$ for manager terraform-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)terragrunt\.hcl$ for manager terragrunt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terragrunt-version$ for manager terragrunt-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^.travis.yml$ for manager travis (repository=nejch/test-renovate-import)
DEBUG: Matched 1 file(s) for manager nvm: .nvmrc (repository=nejch/test-renovate-import)
DEBUG: Found nvm package files (repository=nejch/test-renovate-import)
DEBUG: Found 1 package file(s) (repository=nejch/test-renovate-import)
 INFO: Dependency extraction complete (repository=nejch/test-renovate-import)
       "baseBranch": "main",
       "stats": {
         "managers": {"nvm": {"fileCount": 1, "depCount": 1}},
         "total": {"fileCount": 1, "depCount": 1}
       }
DEBUG: Package releases lookups complete (repository=nejch/test-renovate-import)
       "baseBranch": "main"
DEBUG: branchifyUpgrades (repository=nejch/test-renovate-import)
DEBUG: 2 flattened updates found: node, node (repository=nejch/test-renovate-import)
DEBUG: Returning 2 branch(es) (repository=nejch/test-renovate-import)
DEBUG: Fetching changelog: https://github.com/nodejs/node (8.13.0 -> v8.17.0) (repository=nejch/test-renovate-import)
DEBUG: Fetching changelog: https://github.com/nodejs/node (8.13.0 -> v16.13.0) (repository=nejch/test-renovate-import)
DEBUG: config.repoIsOnboarded=false (repository=nejch/test-renovate-import)
DEBUG: packageFiles with updates (repository=nejch/test-renovate-import)
       "config": {
         "nvm": [
           {
             "packageFile": ".nvmrc",
             "deps": [
               {
                 "depName": "node",
                 "currentValue": "8.13.0",
                 "datasource": "github-tags",
                 "lookupName": "nodejs/node",
                 "depIndex": 0,
                 "updates": [
                   {
                     "bucket": "non-major",
                     "newVersion": "v8.17.0",
                     "newValue": "8.17.0",
                     "releaseTimestamp": "2019-12-18T18:55:49.000Z",
                     "newMajor": 8,
                     "newMinor": 17,
                     "updateType": "minor",
                     "branchName": "renovate/node-8.x"
                   },
                   {
                     "bucket": "major",
                     "newVersion": "v16.13.0",
                     "newValue": "16.13.0",
                     "releaseTimestamp": "2021-10-26T16:22:38.000Z",
                     "newMajor": 16,
                     "newMinor": 13,
                     "updateType": "major",
                     "branchName": "renovate/node-16.x"
                   }
                 ],
                 "warnings": [],
                 "versioning": "node",
                 "sourceUrl": "https://github.com/nodejs/node",
                 "currentVersion": "8.13.0",
                 "isSingleVersion": true,
                 "fixedVersion": "8.13.0"
               }
             ]
           }
         ]
       }
DEBUG: ensureOnboardingPr() (repository=nejch/test-renovate-import)
DEBUG: getBranchPr(renovate/configure) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, open) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, closed) (repository=nejch/test-renovate-import)
DEBUG: Found PR #1 (repository=nejch/test-renovate-import)
DEBUG: Filling in onboarding PR template (repository=nejch/test-renovate-import)
DEBUG: getConfigDesc() (repository=nejch/test-renovate-import)
DEBUG: getDescriptionArray() (repository=nejch/test-renovate-import)
DEBUG: getScheduleDesc() (repository=nejch/test-renovate-import)
DEBUG: No schedule (repository=nejch/test-renovate-import)
DEBUG: No config description found (repository=nejch/test-renovate-import)
DEBUG: getPrList() (repository=nejch/test-renovate-import)
DEBUG: Creating onboarding PR (repository=nejch/test-renovate-import)
DEBUG: Creating PR (repository=nejch/test-renovate-import)
       "title": "Configure (values from config.js)",
       "head": "nejch:renovate/configure",
       "base": "main",
       "draft": false
DEBUG: PR created (repository=nejch/test-renovate-import, branch=renovate/configure)
       "pr": 2,
       "draft": false
DEBUG: Adding labels '' to #2 (repository=nejch/test-renovate-import)
 INFO: Onboarding PR created (repository=nejch/test-renovate-import)
       "pr": "Pull Request #2"
DEBUG: addAssigneesReviewers(pr=2) (repository=nejch/test-renovate-import)
DEBUG: processRepo() (repository=nejch/test-renovate-import)
DEBUG: Closing Dependency Dashboard (repository=nejch/test-renovate-import)
DEBUG: Retrieving issueList (repository=nejch/test-renovate-import)
DEBUG: Retrieved 0 issues (repository=nejch/test-renovate-import)
DEBUG: Removing any stale branches (repository=nejch/test-renovate-import)
DEBUG: config.repoIsOnboarded=false (repository=nejch/test-renovate-import)
DEBUG: Branch lists (repository=nejch/test-renovate-import)
       "branchList": ["renovate/configure"],
       "renovateBranches": ["renovate/configure"]
DEBUG: remainingBranches= (repository=nejch/test-renovate-import)
DEBUG: No branches to clean up (repository=nejch/test-renovate-import)
DEBUG: Repository timing splits (milliseconds) (repository=nejch/test-renovate-import)
       "splits": {"init": 4938, "extract": 299, "lookup": 706, "update": 933},
       "total": 7149
DEBUG: http statistics (repository=nejch/test-renovate-import)
       "urls": {
         "https://api.github.com/graphql (POST)": 3,
         "https://api.github.com/repos/nejch/test-renovate-import/pulls (GET)": 1,
         "https://api.github.com/repos/nejch/test-renovate-import/pulls (POST)": 1
       },
       "hostStats": {"api.github.com": {"requestCount": 5, "requestAvgMs": 374, "queueAvgMs": 0}},
       "totalRequests": 5
 INFO: Repository finished (repository=nejch/test-renovate-import)
       "durationMs": 7149
DEBUG: Renovate exiting
Done in 15.27s.
</pre>
</details>


Run with custom.json and env var, created https://github.com/nejch/test-renovate-import/pull/3:
(`LOG_LEVEL=debug RENOVATE_CONFIG_FILE=custom.json yarn start nejch/test-renovate-import`)
```json
{
  "onboardingPrTitle": "Configure (values from custom.json)",
  "repositories": ["nejch/test-renovate-import"]
}
```
<details>
<pre>
yarn run v1.22.15
$ run-s generate:* 
$ node tools/generate-imports.mjs
generating imports
> data/node-js-schedule.json
> node_modules/emojibase-data/en/shortcodes/github.json
$ node -r ts-node/register/transpile-only -- lib/renovate.ts nejch/test-renovate-import
DEBUG: Using RE2 as regex engine
DEBUG: Parsing configs
DEBUG: Checking for config file in /home/nejc/repos/renovate/custom.json
DEBUG: File config
       "config": {
         "onboardingPrTitle": "Configure (values from custom.json)",
         "repositories": ["nejch/test-renovate-import"]
       }
DEBUG: CLI config
       "config": {"repositories": ["nejch/test-renovate-import"]}
DEBUG: Env config
       "config": {"hostRules": [], "token": "***********"}
DEBUG: Combined config
       "config": {
         "onboardingPrTitle": "Configure (values from custom.json)",
         "repositories": ["nejch/test-renovate-import"],
         "hostRules": [],
         "token": "***********"
       }
DEBUG: Using default github endpoint: https://api.github.com/
DEBUG: GitHub 404
       "url": "https://api.github.com/user/emails"
DEBUG: Cannot read user/emails endpoint on GitHub to retrieve gitAuthor
DEBUG: Platform config
       "platformConfig": {"hostType": "github", "endpoint": "https://api.github.com/", "isGhe": false},
       "renovateUsername": "nejch"
DEBUG: Adding token authentication for api.github.com to hostRules
DEBUG: Using baseDir: /tmp/renovate
DEBUG: Using cacheDir: /tmp/renovate/cache
DEBUG: Initializing Renovate internal cache into /tmp/renovate/cache/renovate/renovate-cache-v1
DEBUG: Commits limit = null
DEBUG: Clearing hostRules
DEBUG: Adding token authentication for api.github.com to hostRules
 INFO: Repository started (repository=nejch/test-renovate-import)
       "renovateVersion": "0.0.0-semantic-release"
DEBUG: Using localDir: /tmp/renovate/repos/github/nejch/test-renovate-import (repository=nejch/test-renovate-import)
DEBUG: initRepo("nejch/test-renovate-import") (repository=nejch/test-renovate-import)
DEBUG: Overriding default GitHub endpoint (repository=nejch/test-renovate-import)
       "endpoint": "https://api.github.com/"
DEBUG: nejch/test-renovate-import default branch = main (repository=nejch/test-renovate-import)
DEBUG: Using personal access token for git init (repository=nejch/test-renovate-import)
DEBUG: resetMemCache() (repository=nejch/test-renovate-import)
DEBUG: Resetting npmrc (repository=nejch/test-renovate-import)
DEBUG: detectSemanticCommits() (repository=nejch/test-renovate-import)
DEBUG: Initializing git repository into /tmp/renovate/repos/github/nejch/test-renovate-import (repository=nejch/test-renovate-import)
DEBUG: Performing blobless clone (repository=nejch/test-renovate-import)
DEBUG: git clone completed (repository=nejch/test-renovate-import)
       "durationMs": 962
DEBUG: latest repository commit (repository=nejch/test-renovate-import)
       "latestCommit": {
         "hash": "ec115fabd73f2789261fc70894175faa98f40dfd",
         "date": "2021-11-15T00:02:24+01:00",
         "message": "Create .nvmrc",
         "refs": "HEAD -> main, origin/main, origin/HEAD",
         "body": "",
         "author_name": "Nejc Habjan",
         "author_email": "hab.nejc@gmail.com"
       }
DEBUG: getCommitMessages (repository=nejch/test-renovate-import)
DEBUG: Semantic commits detection: unknown (repository=nejch/test-renovate-import)
DEBUG: No semantic commits detected (repository=nejch/test-renovate-import)
DEBUG: checkOnboarding() (repository=nejch/test-renovate-import)
DEBUG: isOnboarded() (repository=nejch/test-renovate-import)
DEBUG: findFile(renovate.json) (repository=nejch/test-renovate-import)
DEBUG: findFile(renovate.json5) (repository=nejch/test-renovate-import)
DEBUG: findFile(.github/renovate.json) (repository=nejch/test-renovate-import)
DEBUG: findFile(.github/renovate.json5) (repository=nejch/test-renovate-import)
DEBUG: findFile(.gitlab/renovate.json) (repository=nejch/test-renovate-import)
DEBUG: findFile(.gitlab/renovate.json5) (repository=nejch/test-renovate-import)
DEBUG: findFile(.renovaterc) (repository=nejch/test-renovate-import)
DEBUG: findFile(.renovaterc.json) (repository=nejch/test-renovate-import)
DEBUG: config file not found (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, Configure (values from custom.json), !open) (repository=nejch/test-renovate-import)
DEBUG: Retrieving PR list (repository=nejch/test-renovate-import)
DEBUG: Retrieved 2 Pull Requests (repository=nejch/test-renovate-import)
DEBUG: Found no closed onboarding PR (repository=nejch/test-renovate-import)
DEBUG: Repo is not onboarded (repository=nejch/test-renovate-import)
DEBUG: getBranchPr(renovate/configure) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, open) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, closed) (repository=nejch/test-renovate-import)
DEBUG: Found PR #2 (repository=nejch/test-renovate-import)
DEBUG: Onboarding PR does not exist (repository=nejch/test-renovate-import)
DEBUG: Checking if this org/owner has a default Renovate preset which can be used. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
DEBUG: Failed to retrieve default.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
 INFO: Fallback to renovate.json file as a preset is deprecated, please use a default.json file instead. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: Failed to retrieve renovate.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: Failed to retrieve renovate-config.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: No default org/owner preset found, so the default onboarding config will be used instead. Note: do not be concerned with any 404 messages that preceded this. (repository=nejch/test-renovate-import)
DEBUG: onboarding config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: No renovate config file found (repository=nejch/test-renovate-import)
DEBUG: migrateAndValidate() (repository=nejch/test-renovate-import)
DEBUG: No config migration necessary (repository=nejch/test-renovate-import)
DEBUG: massaged config (repository=nejch/test-renovate-import)
       "config": {}
DEBUG: migrated config (repository=nejch/test-renovate-import)
       "config": {}
DEBUG: Found repo ignorePaths (repository=nejch/test-renovate-import)
       "ignorePaths": ["**/node_modules/**", "**/bower_components/**"]
DEBUG: Using file match: (^|/)tasks/[^/]+\.ya?ml$ for manager ansible (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)galaxy\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: azure.*pipelines?.*\.ya?ml$ for manager azure-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect(-bundle)?\.yml$ for manager batect (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect$ for manager batect-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)WORKSPACE(|\.bazel)$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.bzl$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.?bitbucket-pipelines\.ya?ml$ for manager bitbucket-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: buildkite\.ya?ml for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.buildkite/.+\.ya?ml$ for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Gemfile$ for manager bundler (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.cake$ for manager cake (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Cargo.toml$ for manager cargo (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).circleci/config.yml$ for manager circleci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)cloudbuild.ya?ml for manager cloudbuild (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Podfile$ for manager cocoapods (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)composer.json$ for manager composer (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)deps\.edn$ for manager deps-edn (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)docker-compose[^/]*\.ya?ml$ for manager docker-compose (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/|\.)Dockerfile$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Dockerfile\.[^/]*$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).drone.yml$ for manager droneci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).gitmodules$ for manager git-submodules (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^(workflow-templates|\.github\/workflows)\/[^/]+\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)action\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci-include (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)go.mod$ for manager gomod (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gradle(\.kts)?$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\.properties$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\/.+\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.versions\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)gradle/wrapper/gradle-wrapper.properties$ for manager gradle-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.yaml$ for manager helm-requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)values.yaml$ for manager helm-values (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)helmfile.yaml$ for manager helmfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Chart.yaml$ for manager helmv3 (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^Formula/[^/]+[.]rb$ for manager homebrew (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.html?$ for manager html (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)plugins\.(txt|ya?ml)$ for manager jenkins (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)kustomization\.yaml for manager kustomize (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)project\.clj$ for manager leiningen (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.js$ for manager meteor (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)mix\.exs$ for manager mix (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).node-version$ for manager nodenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.json$ for manager npm (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:cs|fs|vb)proj$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:props|targets)$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)dotnet-tools\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)global\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.nvmrc$ for manager nvm (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)requirements\.(txt|pip)$ for manager pip_requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup.py$ for manager pip_setup (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Pipfile$ for manager pipenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pyproject\.toml$ for manager poetry (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.pre-commit-config\.yaml$ for manager pre-commit (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pubspec\.ya?ml$ for manager pub (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).python-version$ for manager pyenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.ruby-version$ for manager ruby-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.sbt$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: project/[^/]*.scala$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup\.cfg$ for manager setup-cfg (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Package\.swift for manager swift (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.tf$ for manager terraform (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terraform-version$ for manager terraform-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)terragrunt\.hcl$ for manager terragrunt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terragrunt-version$ for manager terragrunt-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^.travis.yml$ for manager travis (repository=nejch/test-renovate-import)
DEBUG: Matched 1 file(s) for manager nvm: .nvmrc (repository=nejch/test-renovate-import)
DEBUG: Found nvm package files (repository=nejch/test-renovate-import)
DEBUG: Found 1 package file(s) (repository=nejch/test-renovate-import)
DEBUG: Need to create onboarding PR (repository=nejch/test-renovate-import)
DEBUG: createOnboardingBranch() (repository=nejch/test-renovate-import)
DEBUG: Checking if this org/owner has a default Renovate preset which can be used. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
DEBUG: Failed to retrieve default.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/default.json"
 INFO: Fallback to renovate.json file as a preset is deprecated, please use a default.json file instead. (repository=nejch/test-renovate-import)
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: Failed to retrieve renovate.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/renovate-config/contents/renovate.json"
DEBUG: GitHub 404 (repository=nejch/test-renovate-import)
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: Failed to retrieve renovate-config.json from repo (repository=nejch/test-renovate-import)
       "statusCode": 404,
       "url": "https://api.github.com/repos/nejch/.github/contents/renovate-config.json"
DEBUG: No default org/owner preset found, so the default onboarding config will be used instead. Note: do not be concerned with any 404 messages that preceded this. (repository=nejch/test-renovate-import)
DEBUG: onboarding config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: Creating onboarding branch (repository=nejch/test-renovate-import)
DEBUG: Committing files to branch renovate/configure (repository=nejch/test-renovate-import)
DEBUG: Setting git author name (repository=nejch/test-renovate-import)
       "gitAuthorName": "Renovate Bot"
DEBUG: Setting git author email (repository=nejch/test-renovate-import)
       "gitAuthorEmail": "renovate@whitesourcesoftware.com"
DEBUG: git commit (repository=nejch/test-renovate-import)
       "deletedFiles": [],
       "ignoredFiles": [],
       "result": {
         "author": null,
         "branch": "renovate/configure",
         "commit": "85179aa",
         "root": false,
         "summary": {"changes": 1, "insertions": 3, "deletions": 0}
       }
DEBUG: No file changes detected. Skipping commit (repository=nejch/test-renovate-import)
       "branchName": "renovate/configure",
       "deletedFiles": [],
       "addedModifiedFiles": ["renovate.json"],
       "ignoredFiles": []
DEBUG: Setting current branch to renovate/configure (repository=nejch/test-renovate-import)
DEBUG: latest commit (repository=nejch/test-renovate-import)
       "branchName": "renovate/configure",
       "latestCommitDate": "2021-11-15T00:18:16+01:00"
DEBUG: Found renovate.json config file (repository=nejch/test-renovate-import)
DEBUG: Repository config (repository=nejch/test-renovate-import)
       "fileName": "renovate.json",
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: migrateAndValidate() (repository=nejch/test-renovate-import)
DEBUG: No config migration necessary (repository=nejch/test-renovate-import)
DEBUG: massaged config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: migrated config (repository=nejch/test-renovate-import)
       "config": {"$schema": "https://docs.renovatebot.com/renovate-schema.json"}
DEBUG: Found repo ignorePaths (repository=nejch/test-renovate-import)
       "ignorePaths": ["**/node_modules/**", "**/bower_components/**"]
DEBUG: No vulnerability alerts found (repository=nejch/test-renovate-import)
DEBUG: No vulnerability alerts found (repository=nejch/test-renovate-import)
DEBUG: No baseBranches (repository=nejch/test-renovate-import)
DEBUG: extract() (repository=nejch/test-renovate-import)
DEBUG: Setting current branch to main (repository=nejch/test-renovate-import)
DEBUG: latest commit (repository=nejch/test-renovate-import)
       "branchName": "main",
       "latestCommitDate": "2021-11-15T00:02:24+01:00"
DEBUG: Using file match: (^|/)tasks/[^/]+\.ya?ml$ for manager ansible (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)galaxy\.ya?ml$ for manager ansible-galaxy (repository=nejch/test-renovate-import)
DEBUG: Using file match: azure.*pipelines?.*\.ya?ml$ for manager azure-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect(-bundle)?\.yml$ for manager batect (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)batect$ for manager batect-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)WORKSPACE(|\.bazel)$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.bzl$ for manager bazel (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.?bitbucket-pipelines\.ya?ml$ for manager bitbucket-pipelines (repository=nejch/test-renovate-import)
DEBUG: Using file match: buildkite\.ya?ml for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.buildkite/.+\.ya?ml$ for manager buildkite (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Gemfile$ for manager bundler (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.cake$ for manager cake (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Cargo.toml$ for manager cargo (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).circleci/config.yml$ for manager circleci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)cloudbuild.ya?ml for manager cloudbuild (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Podfile$ for manager cocoapods (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)composer.json$ for manager composer (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)deps\.edn$ for manager deps-edn (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)docker-compose[^/]*\.ya?ml$ for manager docker-compose (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/|\.)Dockerfile$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Dockerfile\.[^/]*$ for manager dockerfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).drone.yml$ for manager droneci (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).gitmodules$ for manager git-submodules (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^(workflow-templates|\.github\/workflows)\/[^/]+\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)action\.ya?ml$ for manager github-actions (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gitlab-ci\.yml$ for manager gitlabci-include (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)go.mod$ for manager gomod (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.gradle(\.kts)?$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\.properties$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)gradle\/.+\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.versions\.toml$ for manager gradle (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)gradle/wrapper/gradle-wrapper.properties$ for manager gradle-wrapper (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)requirements\.yaml$ for manager helm-requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)values.yaml$ for manager helm-values (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)helmfile.yaml$ for manager helmfile (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Chart.yaml$ for manager helmv3 (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^Formula/[^/]+[.]rb$ for manager homebrew (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.html?$ for manager html (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)plugins\.(txt|ya?ml)$ for manager jenkins (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)kustomization\.yaml for manager kustomize (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)project\.clj$ for manager leiningen (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pom\.xml$ for manager maven (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.js$ for manager meteor (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)mix\.exs$ for manager mix (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).node-version$ for manager nodenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)package.json$ for manager npm (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:cs|fs|vb)proj$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.(?:props|targets)$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)dotnet-tools\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|\/)global\.json$ for manager nuget (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.nvmrc$ for manager nvm (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)([\w-]*)requirements\.(txt|pip)$ for manager pip_requirements (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup.py$ for manager pip_setup (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Pipfile$ for manager pipenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pyproject\.toml$ for manager poetry (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.pre-commit-config\.yaml$ for manager pre-commit (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)pubspec\.ya?ml$ for manager pub (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/).python-version$ for manager pyenv (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.ruby-version$ for manager ruby-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.sbt$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: project/[^/]*.scala$ for manager sbt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)setup\.cfg$ for manager setup-cfg (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)Package\.swift for manager swift (repository=nejch/test-renovate-import)
DEBUG: Using file match: \.tf$ for manager terraform (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terraform-version$ for manager terraform-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)terragrunt\.hcl$ for manager terragrunt (repository=nejch/test-renovate-import)
DEBUG: Using file match: (^|/)\.terragrunt-version$ for manager terragrunt-version (repository=nejch/test-renovate-import)
DEBUG: Using file match: ^.travis.yml$ for manager travis (repository=nejch/test-renovate-import)
DEBUG: Matched 1 file(s) for manager nvm: .nvmrc (repository=nejch/test-renovate-import)
DEBUG: Found nvm package files (repository=nejch/test-renovate-import)
DEBUG: Found 1 package file(s) (repository=nejch/test-renovate-import)
 INFO: Dependency extraction complete (repository=nejch/test-renovate-import)
       "baseBranch": "main",
       "stats": {
         "managers": {"nvm": {"fileCount": 1, "depCount": 1}},
         "total": {"fileCount": 1, "depCount": 1}
       }
DEBUG: Package releases lookups complete (repository=nejch/test-renovate-import)
       "baseBranch": "main"
DEBUG: branchifyUpgrades (repository=nejch/test-renovate-import)
DEBUG: 2 flattened updates found: node, node (repository=nejch/test-renovate-import)
DEBUG: Returning 2 branch(es) (repository=nejch/test-renovate-import)
DEBUG: Fetching changelog: https://github.com/nodejs/node (8.13.0 -> v8.17.0) (repository=nejch/test-renovate-import)
DEBUG: Fetching changelog: https://github.com/nodejs/node (8.13.0 -> v16.13.0) (repository=nejch/test-renovate-import)
DEBUG: config.repoIsOnboarded=false (repository=nejch/test-renovate-import)
DEBUG: packageFiles with updates (repository=nejch/test-renovate-import)
       "config": {
         "nvm": [
           {
             "packageFile": ".nvmrc",
             "deps": [
               {
                 "depName": "node",
                 "currentValue": "8.13.0",
                 "datasource": "github-tags",
                 "lookupName": "nodejs/node",
                 "depIndex": 0,
                 "updates": [
                   {
                     "bucket": "non-major",
                     "newVersion": "v8.17.0",
                     "newValue": "8.17.0",
                     "releaseTimestamp": "2019-12-18T18:55:49.000Z",
                     "newMajor": 8,
                     "newMinor": 17,
                     "updateType": "minor",
                     "branchName": "renovate/node-8.x"
                   },
                   {
                     "bucket": "major",
                     "newVersion": "v16.13.0",
                     "newValue": "16.13.0",
                     "releaseTimestamp": "2021-10-26T16:22:38.000Z",
                     "newMajor": 16,
                     "newMinor": 13,
                     "updateType": "major",
                     "branchName": "renovate/node-16.x"
                   }
                 ],
                 "warnings": [],
                 "versioning": "node",
                 "sourceUrl": "https://github.com/nodejs/node",
                 "currentVersion": "8.13.0",
                 "isSingleVersion": true,
                 "fixedVersion": "8.13.0"
               }
             ]
           }
         ]
       }
DEBUG: ensureOnboardingPr() (repository=nejch/test-renovate-import)
DEBUG: getBranchPr(renovate/configure) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, open) (repository=nejch/test-renovate-import)
DEBUG: findPr(renovate/configure, undefined, closed) (repository=nejch/test-renovate-import)
DEBUG: Found PR #2 (repository=nejch/test-renovate-import)
DEBUG: Filling in onboarding PR template (repository=nejch/test-renovate-import)
DEBUG: getConfigDesc() (repository=nejch/test-renovate-import)
DEBUG: getDescriptionArray() (repository=nejch/test-renovate-import)
DEBUG: getScheduleDesc() (repository=nejch/test-renovate-import)
DEBUG: No schedule (repository=nejch/test-renovate-import)
DEBUG: No config description found (repository=nejch/test-renovate-import)
DEBUG: getPrList() (repository=nejch/test-renovate-import)
DEBUG: Creating onboarding PR (repository=nejch/test-renovate-import)
DEBUG: Creating PR (repository=nejch/test-renovate-import)
       "title": "Configure (values from custom.json)",
       "head": "nejch:renovate/configure",
       "base": "main",
       "draft": false
DEBUG: PR created (repository=nejch/test-renovate-import, branch=renovate/configure)
       "pr": 3,
       "draft": false
DEBUG: Adding labels '' to #3 (repository=nejch/test-renovate-import)
 INFO: Onboarding PR created (repository=nejch/test-renovate-import)
       "pr": "Pull Request #3"
DEBUG: addAssigneesReviewers(pr=3) (repository=nejch/test-renovate-import)
DEBUG: processRepo() (repository=nejch/test-renovate-import)
DEBUG: Closing Dependency Dashboard (repository=nejch/test-renovate-import)
DEBUG: Retrieving issueList (repository=nejch/test-renovate-import)
DEBUG: Retrieved 0 issues (repository=nejch/test-renovate-import)
DEBUG: Removing any stale branches (repository=nejch/test-renovate-import)
DEBUG: config.repoIsOnboarded=false (repository=nejch/test-renovate-import)
DEBUG: Branch lists (repository=nejch/test-renovate-import)
       "branchList": ["renovate/configure"],
       "renovateBranches": ["renovate/configure"]
DEBUG: remainingBranches= (repository=nejch/test-renovate-import)
DEBUG: No branches to clean up (repository=nejch/test-renovate-import)
DEBUG: Repository timing splits (milliseconds) (repository=nejch/test-renovate-import)
       "splits": {"init": 4792, "extract": 297, "lookup": 3481, "update": 2144},
       "total": 11012
DEBUG: http statistics (repository=nejch/test-renovate-import)
       "urls": {
         "https://api.github.com/graphql (POST)": 3,
         "https://api.github.com/repos/nejch/test-renovate-import/pulls (GET)": 1,
         "https://api.github.com/repos/nejch/test-renovate-import/pulls (POST)": 1,
         "https://api.github.com/repos/nodejs/node/releases (GET)": 1,
         "https://api.github.com/repos/nodejs/node/tags (GET)": 1,
         "https://api.github.com/repositories/27193779/releases (GET)": 2,
         "https://api.github.com/repositories/27193779/tags (GET)": 7
       },
       "hostStats": {"api.github.com": {"requestCount": 16, "requestAvgMs": 507, "queueAvgMs": 0}},
       "totalRequests": 16
 INFO: Repository finished (repository=nejch/test-renovate-import)
       "durationMs": 11012
DEBUG: Renovate exiting
Done in 19.81s.
</pre>
</details>

Reply:
`config.js` didn't work:

```
DEBUG: No config file found on disk - skipping
DEBUG: File config
       "config": {}
```

Reply:
i think we need something like this:
```suggestion
    const tmpConfig = await import(configFile);
    config = tmpConfig.default ? tmpConfig.default : tmpConfig;
```

That way we should support old common js exports and newer `export default ...` later

Reply:
@viceice sorry just to check again, how did you run that? Could you paste a few more lines for context from before that debug?

I ran this straight from the repo after `yarn build`, using `yarn start`.

First I ran it without a `config.js` (first log). This produced the empty file config as well for me.

But once I added `config.js` to the root of the repo it did pick it up (second log):
```
DEBUG: Checking for config file in /home/nejc/repos/renovate/config
DEBUG: File config
       "config": {
         "onboardingPrTitle": "Configure (values from config.js)",
         "repositories": ["nejch/test-renovate-import"]
       }
```

Reply:
It's the relevant part from you logs posted above.

Reply:
Ok, thanks for the suggestion I'll apply this!

I'm still confused about the `config.js` not working because in that run where I had `DEBUG: Checking for config file in /home/nejc/repos/renovate/config
DEBUG: No config file found on disk - skipping` happened, the file _really_ did not exist. I added the file only for the 2nd run (that log has correct values from the file). But maybe I am missing something and the bot should read from some default `config.js` that always exists in the image or so?

Reply:
![image](https://user-images.githubusercontent.com/1798109/141976664-b313fd74-e5c0-42d6-9ccf-003d8601fcb8.png)
That's why it works for you 🙈  typescript magic

![image](https://user-images.githubusercontent.com/1798109/141976728-1c20c030-b136-4d2e-aca4-b911090ef7c9.png)

so for later esm compatibillity (where that magic don't exist, we should fallback to full object if `default` isn't defined.

Reply:
Thanks @viceice I don't quite know the internals and transpiling magic :D. I've applied this. Should we do some more testing now any next steps?